src/processor.ts

```diff
@@ -12,21 +12,21 @@ import { lookupArchive } from "@subsquid/archive-registry";
 import { events } from "./types";
 
 const blockRange = { from: parseInt(process.env.BLOCK_RANGE!, 10) };
-const archive =
-  process.env.ARCHIVE != ""
-    ? lookupArchive(process.env.ARCHIVE!, {
-        type: "Substrate",
-        release: "ArrowSquid",
-      })
-    : undefined;
-console.log(`Archive: ${archive}`);
+// const archive =
```

Comment:
Also remove if not used. Do the same for all commented code

Reply:
This one, I want to keep.  in theory this should work. But something is broken on the archive side with subsquid.  I will address it with them, then uncomment this code.